From d267979af9524372f00a0759f02e75c60f604da5 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date: Mon, 15 Oct 2018 13:08:09 +0200
Subject: [PATCH] krita_qimageio.desktop: Fix MimeType separators

Summary:
desktop file did not pass validation.

/usr/share/applications/krita_qimageio.desktop: error: (will be fatal in the future): value "image/bmp,image/x-xpixmap,image/x-xbitmap,image/webp,image/vnd.microsoft.icon" for key
"MimeType" in group "Desktop Entry" contains value "image/bmp,image/x-xpixmap,image/x-xbitmap,image/webp,image/vnd.microsoft.icon" which is an invalid MIME type:
"image/bmp,image/x-xpixmap,image/x-xbitmap,image/webp,image/vnd.microsoft.icon" contains an invalid character in the subtype

Reviewers: #krita, rempt

Reviewed By: #krita, rempt

Tags: #krita

Differential Revision: https://phabricator.kde.org/D16222
---
 plugins/impex/qimageio/krita_qimageio.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/impex/qimageio/krita_qimageio.desktop b/plugins/impex/qimageio/krita_qimageio.desktop
index 4d7dabe235..af366fef07 100644
--- a/plugins/impex/qimageio/krita_qimageio.desktop
+++ b/plugins/impex/qimageio/krita_qimageio.desktop
@@ -63,7 +63,7 @@ Name[x-test]=xxKritaxx
 Name[zh_CN]=Krita
 Name[zh_TW]=Krita
 Exec=krita %f
-MimeType=image/bmp,image/x-xpixmap,image/x-xbitmap,image/webp,image/vnd.microsoft.icon
+MimeType=image/bmp;image/x-xpixmap;image/x-xbitmap;image/webp;image/vnd.microsoft.icon;
 Type=Application
 Icon=calligrakrita
 Categories=Qt;KDE;Office;Graphics;
-- 
2.18.0

